partner_external_map/models/res_partner.py

```diff
@@ -26,6 +26,8 @@ def _address_as_string(self):
             addr.append(self.city)
         if self.state_id:
             addr.append(self.state_id.name)
```

Comment:
These three conditionals are untested, which is why Codecov is griping. 

The solution here would be to
* update the tests to include the three attributes when creating [the test record](https://github.com/george-mengelberg/partner-contact/blob/5ccc17b06c4fc455142c0db4811e200f64ac2076/partner_external_map/tests/test_partner_external_map.py#L22). 
* update the [expected URL](https://github.com/george-mengelberg/partner-contact/blob/5ccc17b06c4fc455142c0db4811e200f64ac2076/partner_external_map/tests/test_partner_external_map.py#L47) to look how it would with the attributes

I won't block though, because the lines you are emulating are also untested. I would love it if you tested your part though, and bonus points if you add the street and street2 attributes at the same time to cover the little bit of lacking coverage here.

Tests are insanely important IMO, so well worth the effort.